.github/workflows/test-full-sync.yml

```diff
@@ -181,21 +158,24 @@ jobs:
         run: |
           SYNC_HEIGHT=""
 
-          while [[ ${SYNC_HEIGHT} == "" ]]; do
-              SYNC_HEIGHT=$(gcloud logging read --format='value(jsonPayload.MESSAGE)' --order="desc" --limit=1 '(resource.labels.instance_id="${{ env.INSTANCE_ID }}" AND jsonPayload.message=~".+finished initial sync to chain tip.+Height\([0-9]+\).+")' | grep -oE 'Height\([0-9]+\)' | grep -oE '[0-9]+' || [[ $? == 1 ]] )
-              echo "SYNC_HEIGHT: $SYNC_HEIGHT"
-              sleep 10
-          done
+          DOCKER_LOGS=$(\
+          gcloud compute ssh \
+          full-sync-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }} \
+          --zone ${{ env.ZONE }} \
+          --quiet \
+          --ssh-flag="-o ServerAliveInterval=5" \
+          --command="docker logs full-sync --tail 20")
+
+          SYNC_HEIGHT=$(echo $DOCKER_LOGS | grep -oE 'finished initial sync to chain tip, using gossiped blocks sync_percent=100.000 % current_height=Height\([0-9]+\)' | grep -oE '[0-9]+' | tail -1 || [[ $? == 1 ]])
 
-          echo "SYNC_HEIGHT=$SYNC_HEIGHT" >> $GITHUB_ENV
```

Comment:
Possible Bug: Does the `SYNC_HEIGHT` environmental variable carry through to the next step without `>> $GITHUB_ENV`?

Reply:
Oops: [fix(ci): revert wrong deletion](https://github.com/ZcashFoundation/zebra/pull/4252/commits/dc0bb66f6e772ee263b0f13d7c766b4d976e43c8)